microsoft-edge/progressive-web-apps-chromium/how-to/offline.md

```diff
@@ -1,183 +1,155 @@
 ---
-title: Offline and network connectivity support in Progressive Web Apps
-description: How to use supporting technologies to create resilient experiences to cater for different network conditions.
+title: Store data on the device
+description: How to use the different data storage options in Progressive Web Apps to create resilient experiences.
 author: MSEdgeTeam
 ms.author: msedgedevrel
 ms.topic: conceptual
 ms.prod: microsoft-edge
 ms.technology: pwa
-ms.date: 01/07/2021
+ms.date: 12/02/2022
 ---
-# Offline and network connectivity support in Progressive Web Apps
+# Store data on the device
 
-For many years, organizations were reluctant to invest heavily in web-based software over native software, because web applications depended on stable network connections. Today, the Progressive Web Apps (PWA) platform offers robust options that enable users to continue working even if the network connection becomes unstable or goes offline.
+Progressive Web Apps (PWA) offer robust options to store data locally to enable users to continue working even if the network connection becomes unstable or goes offline.
+
+There are several ways in which a PWA can store data on a device, such as local storage, Cache API, or IndexedDB.
+
+The following table describes the different options, and the rest of this article goes into more details and usage scenarios for each option.
+
+| Storage option | Description |
+|:--- |:--- |
+| Web Storage | Web Storage has two types: session and local. Web Storage is useful to store small amounts of data from your app's front-end code. The data is structured as key-value pairs and is only available to the current app origin. In the case of session storage, the data is cleared when the session ends, for example when the app is closed, or when the user browses to another origin in the same window or tab. Local storage persists until the app removes the data. |
+| IndexedDB | IndexedDB is an API for storing larger amounts of structured data. The API is asynchronous and can be used both from your app's front-end code and service worker code. Use the IndexedDB API to store a significant amount of structured data on the client, or binary data, such as encrypted media objects or files. |
+| Cache | The Cache API can be used to manage cached resources. The Cache API is Promise-based and allows developers to store and retrieve many web resources—HTML, CSS, JavaScript, images, JSON, and so on. Usually, the Cache API is used within the context of a service worker, but it's also available to your app's front-end code. |
+| File System Access | The File System Access API allows your PWA to read files and folders on the user's device and save changes back to them. |
+
+**Note**: do not use WebSQL or Application Cache. While these are two other browser storage mechanisms, they have both been deprecated. Instead of WebSQL, use IndexedDB. Instead of Application Cache, use the Cache API.
```

Comment:
```suggestion
**Note**: Do not use WebSQL or Application Cache. While these are two other browser storage mechanisms, they have both been deprecated. Instead of WebSQL, use IndexedDB. Instead of Application Cache, use the Cache API.
```